### ISSUE TITLE
redist.group.percent is off when plans are 0-indexed

### ISSUE THREAD

**Issue by kuriwaki** [alarm-redist/redist#52]
The reprex below should give me 3 rows, one per district, but it only gives 2. 

group.percent is missing the 0-index.  Changing
https://github.com/kosukeimai/redist/blob/11bf768fce4c4609b7212c8e24c648c7df75111e/R/group_percent.R#L65
to something like `length(unique(plans[, 1))` should fix the bug. 

``` r
library(redist)
#> 
#> Attaching package: 'redist'
#> The following object is masked from 'package:stats':
#> 
#>     filter
data(fl25)
data(fl25_enum)
data(fl25_adj)

## example code - 
init_plan <- fl25_enum$plans[, 5118]
fit_mcmc <- redist.mcmc(
  adj = fl25_adj, total_pop = fl25$pop,
  init_plan = init_plan, nsims = 100)
#> 
#> ==================== 
#> redist.mcmc(): Automated Redistricting Simulation Using
#>          Markov Chain Monte Carlo
#> 
#> Preprocessing data.
#> 
#> Starting swMH().
#> 10 percent done.
#> Metropolis acceptance ratio: 1.11111
#> 
#> 20 percent done.
#> Metropolis acceptance ratio: 1
#> 
#> 30 percent done.
#> Metropolis acceptance ratio: 1
#> 
#> 40 percent done.
#> Metropolis acceptance ratio: 0.974359
#> 
#> 50 percent done.
#> Metropolis acceptance ratio: 0.979592
#> 
#> 60 percent done.
#> Metropolis acceptance ratio: 0.966102
#> 
#> 70 percent done.
#> Metropolis acceptance ratio: 0.971014
#> 
#> 80 percent done.
#> Metropolis acceptance ratio: 0.974684
#> 
#> 90 percent done.
#> Metropolis acceptance ratio: 0.966292
#> 
#> 100 percent done.
#> Metropolis acceptance ratio: 0.969697

# get stats by district by iter
pct_mcmc <- redist.group.percent(
  plans = fit_mcmc$plans,
  group_pop = fl25$BlackPop,
  total_pop = fl25$TotPop)


# should be 3 rows
dim(pct_mcmc)
#> [1]   2 100
stopifnot(nrow(pct_mcmc) == length(unique(init_plan)))
#> Error: nrow(pct_mcmc) == length(unique(init_plan)) is not TRUE
```

<sup>Created on 2021-03-23 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by christopherkenny** [comment]
Good catch @kuriwaki

**Comment by christopherkenny** [comment]
Actually fixed by #3dc4c20
